services/app-api/utils/changeUserStatus.js

```diff
@@ -24,57 +24,79 @@ export const changeUserStatus = async ({
   territory,
   status,
 }) => {
-  // add a new v0 and v latest
+ // add a new v0 and v latest
+  const updateParams = {
+    TableName: process.env.oneMacTableName,
+    ReturnValues: "UPDATED_NEW",
+    Key: {
+      pk: email.toLowerCase(),
+      sk: `v0#${role}#${territory}`,
+    },
+    UpdateExpression:
+      "ADD Latest :incrval SET " +
```

Comment:
Im not sure this is a better implementation than the original handling using if_not_exists. Both implementations seem to be functionally identical so not understanding why the change was needed. Also AWS documentation suggests to use SET over ADD operation: https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Expressions.UpdateExpressions.html#Expressions.UpdateExpressions.ADD